src/Builder/FormContractorInterface.php

```diff
@@ -23,6 +23,13 @@
  */
 interface FormContractorInterface extends BuilderInterface
 {
+    /**
+     * Adds missing information to the given field description and the given admin.
+     *
+     * @param FieldDescriptionInterface $fieldDescription will be modified
```

Comment:
IMO, this description is not clear enough.

Reply:
I copied pasted the existing ond